frontend/ui/src/app/tutorials/registry/registry-tutorial.component.html

```diff
@@ -0,0 +1,343 @@
+                      docker tag ghcr.io/glasskube/hello-distr/backend:0.1.9 {{ host }}/{{
+                        slug
+                      }}/hello-distr/backend:0.1.9
+                    </code>
+                    <p class="mt-2 text-xs">Do the same for hello-distr/proxy and hello-distr/frontend.</p>
+                    @if (usageFormGroup.controls.tagDone.touched && usageFormGroup.controls.tagDone.invalid) {
+                      <div class="mt-2 text-sm text-red-500">
+                        Please mark this as checked when you are done with it.
+                      </div>
+                    }
+                  </li>
+
+                  <li class="mb-10 ms-8">
+                    <span class="absolute flex items-center justify-center -start-2">
+                      <input
+                        type="checkbox"
+                        [formControl]="usageFormGroup.controls.pushDone"
+                        class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
+                    </span>
+                    <h3 class="text-lg font-medium leading-tight">Push tagged images to the Distr registry</h3>
+                    <code class="mt-2 select-all block text-sm text-gray-900 dark:text-gray-200 whitespace-pre-line">
+                      docker push {{ host }}/{{ slug }}/hello-distr/backend:0.1.9
+                    </code>
+                    <p class="mt-2 text-xs">Do the same for hello-distr/proxy and hello-distr/frontend.</p>
+                    @if (usageFormGroup.controls.pushDone.touched && usageFormGroup.controls.pushDone.invalid) {
+                      <div class="mt-2 text-sm text-red-500">
+                        Please mark this as checked when you are done with it.
+                      </div>
+                    }
+                  </li>
+
+                  <li class="mb-10 ms-8">
+                    <span class="absolute flex items-center justify-center -start-2">
+                      <input
+                        type="checkbox"
+                        [formControl]="usageFormGroup.controls.exploreDone"
+                        class="w-4 h-4 text-blue-600 bg-gray-100 border-gray-300 rounded-sm focus:ring-blue-500 dark:focus:ring-blue-600 dark:ring-offset-gray-700 dark:focus:ring-offset-gray-700 focus:ring-2 dark:bg-gray-600 dark:border-gray-500" />
+                    </span>
+                    <h3 class="text-lg font-medium leading-tight">Pull image and view download history</h3>
+                    <p class="mt-1 text-xs">Pull one of the pushed images:</p>
+                    <code class="mt-2 select-all block text-sm text-gray-900 dark:text-gray-200 whitespace-pre-line">
+                      docker pull {{ host }}/{{ slug }}/hello-distr/backend:0.1.9
```

Comment:
Note that this will only verify that all blobs from the manifest are already available locally. to really pull the image you have to do delete the image digest locally (i.e. all tags) like this: `docker image rm b04ff0cd0287 -f`

Reply:
I'm not sure, if the image ID is deterministic we can include it in the tutorial. Maybe in the same step as pull?

If not, I would just leave it out.